src/Draco.Editor.Web/App.cs

```diff
@@ -0,0 +1,183 @@
+using BlazorMonaco;
+using Draco.Compiler.Api.Scripting;
+using ICSharpCode.Decompiler.Disassembler;
+using ICSharpCode.Decompiler.Metadata;
+using ICSharpCode.Decompiler;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System.IO.Compression;
+using System.Text;
+
+namespace Draco.Editor.Web;
+
+public partial class App
+{
+    private MonacoEditor? DracoEditor { get; set; }
+    private MonacoEditor? OutputViewer { get; set; }
+    private string? SelectedOutputType { get; set; } = "Run";
+    private static readonly StandaloneEditorConstructionOptions defaultDracoEditorOptions = new()
```

Comment:
Please put an empty line between the props and method.

Reply:
That's not a method sir.